packages/graphql/src/schema/get-auth.ts

```diff
@@ -21,7 +21,13 @@ import { DirectiveNode, valueFromASTUntyped } from "graphql";
 import { Auth, AuthRule, AuthOperations } from "../types";
 
 const validOperations: AuthOperations[] = ["CREATE", "READ", "UPDATE", "DELETE", "CONNECT", "DISCONNECT"];
-const validFields = ["operations", "AND", "OR", "allow", "where", "bind", "isAuthenticated", "roles"];
+const validFields = [
+    "operations",
+    "AND", "OR",
+    "allow", "where", "bind",
+    "isAuthenticated", "allowUnauthenticated",
+    "roles"
+];
```

Comment:
I don't like this array formatting, surely ESLint/Prettier didn't format it like this?

Reply:
Sure, I just put back the previous formatting.